kafka-java-liberty-sample/src/main/java/com/eventstreams/samples/servlet/RESTRequest.java

```diff
@@ -119,7 +121,6 @@ public String post(String target, String body, int[] ignoredErrorCodes) {
         }
 
         try {
-
             // Create secure connection to the REST URL.
             SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
             sslContext.init(null, null, null);
```

Comment:
You've not made any logic changes to this file, so can you undo the cosmetic changes?